libraries/common/constants/Portals.js

```diff
@@ -66,9 +67,14 @@ export const PAGE_LOGIN_REGISTER_LINK_BEFORE = `${PAGE}.${LOGIN}.${REGISTER_LINK
 export const PAGE_LOGIN_REGISTER_LINK = `${PAGE}.${LOGIN}.${REGISTER_LINK}`;
 export const PAGE_LOGIN_REGISTER_LINK_AFTER = `${PAGE}.${LOGIN}.${REGISTER_LINK}.${AFTER}`;
 
+export const NAV_MENU_CONTENT = `${NAV_MENU}.${CONTENT}`;
 export const NAV_MENU_CONTENT_BEFORE = `${NAV_MENU}.${CONTENT}.${BEFORE}`;
 export const NAV_MENU_CONTENT_AFTER = `${NAV_MENU}.${CONTENT}.${AFTER}`;
 
+export const QUICK_LINKS_CONTENT = `${QUICK_LINKS}.${CONTENT}`;
+export const QUICK_LINKS_CONTENT_BEFORE = `${QUICK_LINKS}.${CONTENT}.${BEFORE}`;
+export const QUICK_LINKS_CONTENT_AFTER = `${QUICK_LINKS}.${CONTENT}.${AFTER}`;
+
```

Comment:
Please name the constants in the same style like the other nav menu constants.
(NAV_MENU_STORE_INFORMATION => NAV_MENU_QUICK_LINKS)